### DISCUSSION TITLE
OverrideElement Color Exploded View

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1307] [Q&A] [answered]
Hello everyone good morning
I'm struggling with emphasizing some elements in my view
through the overrideElements of the EmphasizeElements class
I can color my elements etc. etc. and so far everything is ok.
At this point I would like the emphasized elements to Exploded like in the example https://www.itwinjs.org/sandboxes/iTwinPlatform/Exploded%20View
when I do the expansion the previously highlighted colors disappear I also implemented the emphasis again for this class but it doesn't seem to work Do you have any ideas on how to help me

Thanks for the precious help

**Comment by mayank99** [comment, marked as answer]
There is no `EmphasizeElements` class in iTwinUI. I think you meant to open this issue in [`itwinjs-core` repo](https://github.com/iTwin/itwinjs-core).
